app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,38 @@
+class OrdersController < BaseController
+  before_action :check_logged_in_user
+  before_action :retrieve_user_cart
+  before_action :check_admin_user, only: %i[index toggle_handled]
+  
+  def create
+    order = current_user.orders.build
+
+    @cart.product_carts.each do | product |
```

Comment:
there's no need for spaces when using `|`

Reply:
Thanks!